hack/tools/logcheck/README.md

```diff
@@ -47,6 +47,12 @@ the next section.
 
 Unstructured klog logging calls are flagged as error.
 
+## klog (disabled by default)
```

Comment:
I don't understand the naming scheme taken here. `klog` doesn't really describe what it checks. Let's discuss naming convention in the commit that first introduces checks.

Reply:
Other suggestions are welcome.

One thing that we need to keep in mind regarding naming is double negation: I was trying to be consistent such that "enabled" means "do something extra" and not "skip some check" for all checks.

"contextual" might work here, with meaning being "only contextual calls allowed". But then we also need to rename "unstructured" to "structured" (= "only structured calls allowed").

Reply:
I've switched to that, see https://github.com/kubernetes/klog/compare/afe3a50a2337e958eed75b452dbdd1d0c61892d4..20e85817451b4f6a0e39da28637ceaf6af5eecdd

Reply:
lgtm